### PR TITLE
Feat: Rewards v2

### PR DESCRIPTION
**Pull request by mul53**

### CODE REVIEW

File: src/reducers/screens/home.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { REHYDRATE } from 'redux-persist/lib/constants'
+import get from 'lodash/get'
+import * as staking from '@/actions/staking'
+import { REWARDS_PLATFORMS } from '@/constants'
+
+const initialState = {
+	stakingPlatform: REWARDS_PLATFORMS.UNISWAP
+}
+
+export default (state = initialState, action) => {
```
Comment (leonprou) [inline src/reducers/screens/home.js:10]: the lining here looks wierd

File: src/sagas/staking.js [hunk 2]
```diff
@@ -232,6 +232,10 @@ function * getStakingPeriod ({ stakingContract, networkId }) {
   }
 }
 
+function * selectStakingPlatform ({ stakingPlatform }) {
+  yield put(ac)
```
Comment (leonprou) [inline src/sagas/staking.js:236]: what's `ac`?

Comment (LiorAgnin) [inline src/sagas/staking.js:236]: The `ac` looks like a typo / mistake, @mul53 Can you check?

**Review by LiorAgnin — COMMENTED** [review]
Looks awesome, Great work!

File: src/sagas/staking.js (hunk 2 above)
Comment (mul53) [reply src/sagas/staking.js:236]: sorry that's some unfinished code i need to remove

**Review by leonprou — APPROVED** [review]

**Review by leonprou — APPROVED** [review]

**Pull request merged**